staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -427,6 +427,15 @@ type Firmware struct {
 	// Settings to set the kernel for booting.
 	// +optional
 	KernelBoot *KernelBoot `json:"kernelBoot,omitempty"`
+	// Information that can be set in the ACPI table
+	ACPI *ACPI `json:"acpi,omitempty"`
+}
+
+type ACPI struct {
+	// SliceNameRef should match the volume name of a secret object. The data in the secret should
```

Comment:
```suggestion
	// SlicNameRef should match the volume name of a secret object. The data in the secret should
```

Reply:
Hi @vasiliy-ul , thanks for the quick review.

I thought about referencing the Secret in the `SlicNameRef` as well but I decided to do the easier path first.

I don't have a strong opinion in this so I can change it. Let's just double check if others agree. @alicefr, @xpivarc  what do you think?

Reply:
IMO, pointing to the volume name is fine. If for any reasons we decide to support another type of volume for this, then this mechanism still remains true. Also, kubernetes references the secrets by volume name inside the pod spec. So, I don't think it is too unusual for the user to reference the volume name instead of the secret name.

Reply:
Just to note: changing later the reference from volume to secret will be a breaking API change. I am not pushing with this, but IMHO it is better to come up with a long-term decision when the API is introduced. Altering it after the release may become problematic.

Reply:
I agree that deciding now is better. For the moment, I'm keeping the current code behavior to reference the volume name.